cli/config.py

```diff
@@ -305,16 +323,21 @@ def _merge_dict(self, config):
 
         spark_configuration = config.get('spark_configuration')
         if spark_configuration:
-            self.spark_defaults_conf = self.__convert_to_path(spark_configuration.get('spark_defaults_conf'))
-            self.spark_env_sh = self.__convert_to_path(spark_configuration.get('spark_env_sh'))
-            self.core_site_xml = self.__convert_to_path(spark_configuration.get('core_site_xml'))
-            self.jars = [self.__convert_to_path(jar) for jar in spark_configuration.get('jars')]
+            self.spark_defaults_conf = self.__convert_to_path(
+                spark_configuration.get('spark_defaults_conf'))
```

Comment:
I feel like splits like this make the code harder to read

Reply:
The auto formatter I think format at x number of column

Reply:
I think we need to look into the formatter so we all have the same settings, max column length, etc. So the editor doesn't format

Reply:
ok so I added yapf and changed vscode default to use that for formatting, it seems like autopep8(default) doesn't listen to the max line length rule. I set the max line length to 120(Instead of default 80) when yapf format it reforms a lot of the splitted lines which makes a lot of changes for this PR so I won't format this file this time(just revert those changes).